contracts/exchange/contracts/src/MixinSignatureValidator.sol

```diff
@@ -244,7 +244,17 @@ contract MixinSignatureValidator is
             hash,
             signature
         );
+        bytes32 magicValue = bytes32(bytes4(keccak256("isValidWalletSignature(bytes32,address,bytes)")));
```

Comment:
Can we add a comment with the `bytes4` constants for both of these signature types?